### ISSUE TITLE
Extra space added in forall

### ISSUE THREAD

**Issue by pbrisbin** [lspitzner/brittany#264]
This diff is between code formatted by v0.12.0.0 and v0.12.1.0:

```diff
 -- | Construction of an action from some JSON-encodable
 someBatchUpdateAction
-  :: forall a . (Typeable a, ToJSON a) => a -> SomeBatchUpdateAction
+  :: forall  a . (Typeable a, ToJSON a) => a -> SomeBatchUpdateAction
 someBatchUpdateAction a =
   SomeBatchUpdateAction $ object ["tag" .= show (typeOf a), "contents" .= a]
```

I believe v0.12.0.0 is correct and there's a bug adding the extra space in v0.12.1.0.

**Comment by lspitzner** [comment]
Thanks, this is fixed on master, see 974826f98fa9f08e659dc334edda61bb5c7cfa49.

**Comment by lspitzner** [comment]
fixed in 0.12.1.1
